docker-compose.override.yml

```diff
@@ -13,20 +13,11 @@ services:
         "
     depends_on:
       - storage-emulator
-    env_file:
-      - .env
 
   storage-emulator:
     image: mcr.microsoft.com/azure-storage/azurite
     volumes:
-      - "blobStorage:/data"
+      - "./blobStorage:/data"
```

Comment:
Any particular reason to store the persistent data to the local folder?

Reply:
Personal preference. 
I like having the actual data easily available for browsing, backup and cleanup.
Without having to go sudo into some "/var/lib/docker/volume" path, that may change on different systems.

Reply:
Fair enough, just remember to add the folder in `.gitignore`.